packages/apps/composer-app/vite.config.ts

```diff
@@ -88,6 +91,19 @@ export default defineConfig({
         resolve(__dirname, '../plugins/*/src/**/*.{js,ts,jsx,tsx}'),
       ],
     }),
+    IconsPlugin({
+      symbolPattern:
+        'ph--([a-z]+[a-z-]*)--(bold|duotone|fill|light|regular|thin)',
+      assetPath: (name, variant) =>
+        `${phosphorIconsCore}/${variant}/${name}${
+          variant === 'regular' ? '' : `-${variant}`
+        }.svg`,
+      spritePath: resolve(__dirname, 'public/icons.svg'),
+      contentPaths: [
+        `${resolve(__dirname, '../../..')}/{packages,tools}/**/dist/**/*.{mjs,html}`,
+        `${resolve(__dirname, '../../..')}/{packages,tools}/**/src/**/*.{ts,tsx,js,jsx,css,md,html}`
```

Comment:
I'm guessing src needs to be included in case things haven't been rebuilt when you're in dev mode?